src/test/java/com/softserve/edu/greencity/ui/pages/tipstricks/TipsTricksPage.java

```diff
@@ -105,7 +92,8 @@ public boolean isDisplayedStartHabitCenter() {
     // ButtonBelow 'Start forming a habit'
 
     public WebElement getStartHabitBelow() {
-        return startHabitBelow;
+        return startHabitBelow = driver
+                .findElement(By.cssSelector(".stat-row-content.ng-star-inserted:nth-child(1) > div >button"));
```

Comment:
need to rewrite this selector

Reply:
Change to:
startHabitCenter = driver
                .findElement(By.xpath(".stat-row-content:nth-child(1) .button"));